<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the public 'contao.security.frontend_preview_authenticator' shared service.

include_once \dirname(__DIR__, 4).''.\DIRECTORY_SEPARATOR.'vendor'.\DIRECTORY_SEPARATOR.'contao'.\DIRECTORY_SEPARATOR.'core-bundle'.\DIRECTORY_SEPARATOR.'src'.\DIRECTORY_SEPARATOR.'Security'.\DIRECTORY_SEPARATOR.'Authentication'.\DIRECTORY_SEPARATOR.'FrontendPreviewAuthenticator.php';

return $this->services['contao.security.frontend_preview_authenticator'] = new \Contao\CoreBundle\Security\Authentication\FrontendPreviewAuthenticator(($this->services['security.helper'] ?? $this->getSecurity_HelperService()), ($this->services['session'] ?? $this->getSessionService()), ($this->privates['contao.security.frontend_user_provider'] ?? $this->load('getContao_Security_FrontendUserProviderService.php')), ($this->privates['monolog.logger'] ?? $this->getMonolog_LoggerService()));
